---
float64 latitude
float64 longitude
float64 altitude
